deploy/gke/simple/README.md

```diff
@@ -67,6 +67,171 @@ Since there is no application currently running on the cluster, the collector is
+    - Update the `spec.volumes` section to add a new `configMap` entries
+        ```
+        volumes:
+          - configMap:
+              name: otel-config
+            name: otel-collector-config-vol
+          # Add the following volume configurations
+          - configMap:
+              name: otlp-test-data-metrics # Volume for test metrics data
+            name: test-data-vol-metrics    # This could be changed to anything
+          - configMap:
+              name: otlp-test-data-traces # Volume for test traces data
+            name: test-data-vol-traces    # This could be changed to anything
+          - configMap:
+              name: otlp-test-data-logs   # Volume for test logs data
+            name: test-data-vol-logs      # This could be changed to anything
+        ```  
+    - Update the `spec.containers.volumeMounts` section to configure new volumeMount(s) for the added volumes.
+        ```
+        volumeMounts:
+        - name: otel-collector-config-vol
+          mountPath: /conf
+        # Add the following volumeMount configurations
+        - name: test-data-vol-metrics  # This name should match to the name of volume in spec.volumes
+          mountPath: /mnt/testdata/metrics   # This is the location in the kubernetes environment where the file for configMap for metrics will be mounted
+        - name: test-data-vol-traces  # This name should match to the name of volume in spec.volumes
+          mountPath: /mnt/testdata/traces   # This is the location in the kubernetes environment where the file for configMap for traces will be mounted
+        - name: test-data-vol-logs  # This name should match to the name of volume in spec.volumes
+          mountPath: /mnt/testdata/logs   # This is the location in the kubernetes environment where the file for configMap for logs will be mounted
+        ```
+6. Deploy the cluster using the new deployment manifest - 
+    ```
+    kubectl apply -f manifest.yaml -n $OTEL_NAMESPACE
+    ```
+7. Verify if the telemetry data is now being emitted by using `kubectl logs` - 
+    ```
+    # Get the pod name(s)
+    kubectl get pods -n $OTEL_NAMESPACE
+
+    # Get the logs from a pod
+    kubectl logs <pod_container_name> -n $OTEL_NAMESPACE
+    ```
+
+*If the logs do not show the expected telemetry data, check if the file used to send json data - [testdata.json](./otlp-data/testdata.json) is found in the Kubernetes deployment. Follow steps in [troubleshooting guide](../troubleshooting.md#verify-updates-to-filesconfigmaps-in-kubernetes-cluster) to verify file presence and contents.*
+
+**NOTE: You will need to update the timestamps in [testdata-metrics](./otlp-data/testdata-metrics.json) to be within 24 hours of current time otherwise the metrics will not show up in Google Cloud console's Metrics Explorer. You might also need to update timestamps in other testdata - metrics and logs in case they become old enough to not be recognized by Google Cloud.**
```

Comment:
is there any way we could automatically update these for the users with a Make command or something? Or give the user a time window to look at in their metrics explorer?

Reply:
I think we could use `sed` with `make` to automate this. Sounds like a good idea.